masonry/src/text/selection.rs

```diff
@@ -184,8 +184,16 @@ impl<T: Selectable> TextWithSelection<T> {
                         }
                         _ => Handled::No,
                     },
-                    winit::keyboard::Key::Unidentified(_) => todo!(),
-                    winit::keyboard::Key::Dead(_) => todo!(),
+                    winit::keyboard::Key::Unidentified(key) => {
+                        // TODO: figure out correct behavior
+                        tracing::warn!(?key, "got unidentified key, did not handle");
+                        Handled::No
+                    }
+                    winit::keyboard::Key::Dead(key) => {
```

Comment:
In selection code, there's no need to handle a dead key, so we can just use `Handled::No`